aem/gradle/environment.gradle.kts

```diff
@@ -47,6 +47,26 @@ configure<AemExtension> {
             dependsOn(":develop")
         }
 
+        instanceSatisfy {
+            packages {
+                // https://github.com/Cognifide/gradle-aem-plugin#task-instancesatisfy
+            }
+        }
+
+        instanceProvision {
```

Comment:
@mierzwid what do you think? :)

Reply:
I like it very much. Well described steps, not a custom code.